### PR TITLE
Cameron/update repo

### PR DESCRIPTION
**Pull request by newtoncn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- [Setup](#setup)
+  - 
+
+
+## Setup 
+
+### Install
+
+To install the repository, run the ./setup_environment.sh script. Anytime you want to use the tensorflow api, run 'source subbots_python/bin/activate'.
+
```
Comment (garethellis0) [inline README.md:18]: We should definitly have a note in big obvious text at the top about how to clone this repo. Something like `git clone --recursive repo_url` I think

**Comment by garethellis0** [comment]
I got the following error when running `setup_environment.sh`:
```
./setup_environment.sh: line 50: ./bin/protoc: No such file or directory
```
probably want to change that to `models/research/bin/protoc`

After fixing that I get:
```
/home/gareth/Downloads/Object-Detection-Training/models/research/object_detection/protos/anchor_generator.proto: File does not reside within any path specified using --proto_path (or -I).  You must specify a --proto_path which encompasses this file.  Note that the proto_path must be an exact prefix of the .proto file names -- protoc is too dumb to figure out when two paths (e.g. absolute and relative) are equivalent (it's harder than you think).
```

You may want to do a fresh clone/install/setup of this and make sure it works for you.

**Comment by newtoncn** [comment]
Did you make sure you were on the right branch? The protoc zip should be downloaded into the base folder

**Comment by newtoncn** [comment]
wget -O protobuf.zip https://github.com/google/protobuf/releases/download/v3.0.0/protoc-3.0.0-linux-x86_64.zip

**Comment by newtoncn** [comment]
Or else the protoc file won't be able to find the proto files

**Comment by garethellis0** [comment]
Hmm. I think you right be right (I may have been on `master`). Switching to this branch and running `setup_environment.sh`, I get:
```
...
Saving to: ‘protobuf.zip’

protobuf.zip                    100%[=======================================================>]   1.24M  1.36MB/s    in 0.9s    

2018-10-16 06:09:51 (1.36 MB/s) - ‘protobuf.zip’ saved [1296281/1296281]

Archive:  protobuf.zip
   creating: include/
   creating: include/google/
   creating: include/google/protobuf/
  inflating: include/google/protobuf/struct.proto  
  inflating: include/google/protobuf/type.proto  
  inflating: include/google/protobuf/descriptor.proto  
  inflating: include/google/protobuf/api.proto  
  inflating: include/google/protobuf/empty.proto  
   creating: include/google/protobuf/compiler/
  inflating: include/google/protobuf/compiler/plugin.proto  
  inflating: include/google/protobuf/any.proto  
  inflating: include/google/protobuf/field_mask.proto  
  inflating: include/google/protobuf/wrappers.proto  
  inflating: include/google/protobuf/timestamp.proto  
  inflating: include/google/protobuf/duration.proto  
  inflating: include/google/protobuf/source_context.proto  
   creating: bin/
  inflating: bin/protoc              
  inflating: readme.txt              
object_detection/protos/grid_anchor_generator.proto: File not found.
object_detection/protos/ssd_anchor_generator.proto: File not found.
object_detection/protos/multiscale_anchor_generator.proto: File not found.
models/research/object_detection/protos/anchor_generator.proto: Import "object_detection/protos/grid_anchor_generator.proto" was not found or had errors.
models/research/object_detection/protos/anchor_generator.proto: Import "object_detection/protos/ssd_anchor_generator.proto" was not found or had errors.
models/research/object_detection/protos/anchor_generator.proto: Import "object_detection/protos/multiscale_anchor_generator.proto" was not found or had errors.
models/research/object_detection/protos/anchor_generator.proto:13:5: "GridAnchorGenerator" is not defined.
models/research/object_detection/protos/anchor_generator.proto:14:5: "SsdAnchorGenerator" is not defined.
models/research/object_detection/protos/anchor_generator.proto:15:5: "MultiscaleAnchorGenerator" is not defined.
./setup_environment.sh: line 56:  3032 Illegal instruction     (core dumped) python $CURR_DIR/models/research/object_detection/builders/model_builder_test.py
```

**Comment by newtoncn** [comment]
Hmm, ok. I think there's one argument missing in the protoc calll but that last core dumped error is weird. I'll check it out tonight

**Comment by newtoncn** [comment]
@garethellis0 Should be fixed

**Comment by newtoncn** [comment]
#2 I've also added travis ci to the repo to confirm

**Comment by newtoncn** [comment]
[Setup ubc-subbots tensorflow object detection training repo](https://trello.com/c/shaiEp93/24-setup-ubc-subbots-tensorflow-object-detection-training-repo)

**Comment by garethellis0** [comment]
Oh, good stuff with travis. If that's passing then we should be good.

**Review by garethellis0 — APPROVED** [review]

**Pull request merged**